SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -221,6 +221,12 @@ void USpatialGameInstance::Init()
 {
 	Super::Init();
 
+	if (IsSimulatedPlayer())
+	{
+		GetEngine()->OnTravelFailure().AddUObject(this, &USpatialGameInstance::HandleOnSimulatedPlayerTravelFailure);
```

Comment:
Can we rely on user always using SpatialGameInstance, or would we be better off moving this to the engine?

Reply:
If they want to have a spatial game they must use SpatialGameInstance.